cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -56,8 +58,48 @@ define([
             // It sends `POST` request, because the model doesn't have `id`. In
             // this case, it is considered to be new according to Backbone documentation.
             AjaxHelpers.expectJsonRequest(
-                requests, 'POST', '/course-details', expectedJson
+                requests, 'POST', urlRoot, expectedJson
             );
         });
+
+        it('should save entrance exam course details information correctly', function () {
+            var entrance_exam_minimum_score_pct = '60';
+            var entrance_exam_enabled = 'true';
+
+            var entrance_exam_min_score = this.view.$('#entrance-exam-minimum-score-pct');
+            var entrance_exam_enabled_field = this.view.$('#entrance-exam-enabled');
+
+            var requests = AjaxHelpers.requests(this),
+                expectedJson = $.extend(true, {}, modelData, {
+                    entrance_exam_enabled: entrance_exam_enabled,
+                    entrance_exam_minimum_score_pct: entrance_exam_minimum_score_pct
+                });
+
+            expect(this.view.$('.div-grade-requirements div')).toBeHidden();
+
+            // select the entrance-exam-enabled checkbox. grade requirement section should be visible
+            entrance_exam_enabled_field
+                .attr('checked', entrance_exam_enabled)
+                .trigger('change');
+            expect(this.view.$('.div-grade-requirements div')).toBeVisible();
+
+            //input some invalid values.
```

Comment:
I prefer to have lots of short tests, rather than one large test like this. I'd break out tests such as

    it('should disallow save with an invalid minimum score percentage'
    it('should provide a default value for the minimum score percentage'
    it('hides the entrance exam when the check box is deselected'

Usually I find that if my test needs comments breaking it into sections, then it is better to make each section its own test. It is then much clearer what has failed, and it is easier to run just the failing code.